docs/build/tutorials/platform/subnets/create-a-virtual-machine-vm.md

```diff
@@ -1466,6 +1464,39 @@ It's possible to give an alias for these IDs. For example, we can alias `Timesta
   ]
 }
 ```
+j
+### Installing a VM
+
+AvalancheGo searches for and registers plugins under the `plugins` directory of the [build directory](https://docs.avax.network/build/references/avalanchego-config-flags/#build-directory).
+
+To install the virtual machine onto your node, you need to move the built virtual machine binary under this directory. Virtual machine executable names must be either a full virtual machine ID (encoded in CB58), or a VM alias.
```

Comment:
Can we add that it's also possible to install them by copying to plugin dir "before" node starts up?

Reply:
Added